pkg/scheduler/apis/config/v1beta2/default_plugins.go

```diff
@@ -170,16 +165,20 @@ func mergePluginSet(defaultPluginSet, customPluginSet v1beta2.PluginSet) v1beta2
 			}
 			// The default plugin is explicitly re-configured, update the default plugin accordingly.
 			if customPlugin, ok := enabledCustomPlugins[defaultEnabledPlugin.Name]; ok {
-				klog.InfoS("Defaut plugin is explicitly re-configured and is overriding.", "plugin", defaultEnabledPlugin.Name)
+				klog.InfoS("Default plugin is explicitly re-configured and is overriding.", "plugin", defaultEnabledPlugin.Name)
```

Comment:
nit: `Default plugin is explicitly re-configured; overriding`

Reply:
Done